fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchTokenTest.java

```diff
@@ -140,15 +160,6 @@ public void testSearchToken_code_chained_missing() throws Exception {
         assertSearchDoesntReturnComposition("subject:Basic.missing-code:missing", "false");
     }
 
-    @Test
-    public void testSearchToken_CodeableConcept_chained_missing() throws Exception {
```

Comment:
any particular reason this one got removed?  or it was just redundant/unnecessary?

Reply:
ah, never mind, just relocated.  now i see on line 239 of the new version